### PR TITLE
Model/developer

### LINKED ISSUE #9: How to deal with uID in Developer's model.py?
I'm not sure what else is required when specifying the field uID in model.py in branch: model/developer.

### LINKED ISSUE #10: How to use models.FileField()?
When I worked on model.py, I used this [link](https://wayhome25.github.io/django/2017/03/20/django-ep5-model/
) as a reference to define the fields.

In case of Developer, we decided to have two entities - profile image and portfolio - from the Design Specification.

`
profile_image = models.FileField(
        verbose_name='profile image',
        upload_to="developer/profile_pic", #not sure
    )
`

When the user uploads his profile image, I remember it will be stored in our modu database.
I'm a little lost here... where will profile image be uploaded to?
I'm not so sure about models.FileField() either.
Any ideas?

### LINKED ISSUE #15: How to make the migration successful for moduDeveloper?
For moduDeveloper, I updated models.py, urls.py, validators.py, views.py and installed Pillow to use ImageField.
        HINT: Get Pillow at https://pypi.org/project/Pillow/ or run command "python -m pip install Pillow".

I'm trying to migrate, but I'm having the following errors.

`SystemCheckError: System check identified some issues:
ERRORS:
moduDeveloper.Developer.favorite: (fields.E300) Field defines a relation with model 'Project', which is either not installed, or is abstract.
moduDeveloper.Developer.favorite: (fields.E307) The field moduDeveloper.Developer.favorite was declared with a lazy reference to 'moduDeveloper.project', but app 'moduDeveloper' doesn't provide model **'project'**.
moduDeveloper.Developer.invite: (fields.E300) Field defines a relation with model **'Project'**, which is either not installed, or is abstract.
moduDeveloper.Developer.invite: (fields.E307) The field moduDeveloper.Developer.invite was declared with a lazy reference to 'moduDeveloper.project', but app 'moduDeveloper' doesn't provide model **'project'**.
moduDeveloper.Developer.member_of: (fields.E300) Field defines a relation with model **'Project'**, which is either not installed, or is abstract.
moduDeveloper.Developer.member_of: (fields.E307) The field moduDeveloper.Developer.member_of was declared with a lazy reference to 'moduDeveloper.project', but app 'moduDeveloper' doesn't provide model **'project'**.
moduDeveloper.Developer.proposed_projects: (fields.E300) Field defines a relation with model 'Project', which is either not installed, or is abstract.
moduDeveloper.Developer.proposed_projects: (fields.E307) The field moduDeveloper.Developer.proposed_projects was declared with a lazy reference to 'moduDeveloper.project', but app 'moduDeveloper' doesn't provide model **'project'**.
moduDeveloper.Developer: (models.E015) 'ordering' refers to the nonexistent field, related field, or lookup 'u_ID'.
moduDeveloper.Developer_favorite.project: (fields.E307) The field moduDeveloper.Developer_favorite.project was declared with a lazy reference to 'moduDeveloper.project', but app 'moduDeveloper' doesn't provide model **'project'**.
moduDeveloper.Developer_invite.project: (fields.E307) The field moduDeveloper.Developer_invite.project was declared with a lazy reference to 'moduDeveloper.project', but app 'moduDeveloper' doesn't provide model **'project'**.
moduDeveloper.Developer_member_of.project: (fields.E307) The field moduDeveloper.Developer_member_of.project was declared with a lazy reference to 'moduDeveloper.project', but app 'moduDeveloper' doesn't provide model **'project'.**`

Does this mean that I need moduProject in my local repository to make this migration successful?

### PR DESCRIPTION
**Pull request by jwl317**
Created a new branch _model/developer_ and updated its model.py

### CODE REVIEW

**Comment by JTzzang** [comment]
I just checked.

You need to create __init__.py under moduDeveloper folder first.

What is the difference between Developer.member_of and Developer.project?

**Comment by jwl317** [comment]
> I just checked.
> 
> You need to create **init**.py under moduDeveloper folder first.
> 
> What is the difference between Developer.member_of and Developer.project?

__init__.py is created. Thank you!

Let's discuss the second question together in 9pm meeting.

**Comment by JTzzang** [comment]
Where is Developer name?

**Comment by jwl317** [comment]
> Where is Developer name?

Oh right. Account model is not available. I'm not sure what to do about it too.
Let's discuss it in the meeting too.

File: moduDeveloper/models.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+#Reference: https://wayhome25.github.io/django/2017/03/20/django-ep5-model/
+
+
+# Create your models here.
+class Developer(models.Model):
+    """A typical class defining a model, derived from the Model class."""
+
+    # Fields
+    uID = models.CharField(
```
Comment (ReverseIdeology) [inline moduDeveloper/models.py]: AutoField

Comment (ReverseIdeology) [reply moduDeveloper/models.py]: unique = True 써주세요

File: moduDeveloper/models.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+        primary_key=True,
+        max_length=20,
+    )
+
+    profile_image = models.FileField(
```
Comment (ReverseIdeology) [inline moduDeveloper/models.py]: ImageField 쓰는 게 좋을 것 같음

File: moduDeveloper/models.py [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+    #Developer-Propose-Project
+    project = models.ForeignKey(
+        'Project',
+        on_delete=models.CASCADE,
+        related_name='developers',
+        related_query_name='developer'
+    )
+
+    #Developer-Write-Assessment
```
Comment (ReverseIdeology) [inline moduDeveloper/models.py]: Assessment 쪽에서 참조하도록 수정할 것

**Pull request merged**